src/WatchDogs.Contracts/WatchDogs.Contracts.csproj

```diff
@@ -8,6 +8,11 @@
 
   <ItemGroup>
     <PackageReference Include="Bogus" Version="35.6.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.8" />
```

Comment:
Why are any of these in contracts? They are not needed here.

Reply:
Removed.